### DISCUSSION TITLE
How does one run Taskfile on Windows?

### DISCUSSION THREAD

**Discussion by eddgrant** [go-task/task#628] [Q&A] [answered]
Hi folks,

Apologies for the most basic of questions, I'm keen to understand how to run Taskfile on a Windows host? I thought it might simply be a case of opening up a Dos prompt and typing `.\task` but Windows gives me the following error when I try that:

 ```Dos
Y:\technical-reference-guide>.\task
'.\task' is not recognized as an internal or external command,
operable program or batch file.
```

I've checked the docs but I can't see anything to point me in the right direction, so would be extremely grateful if someone could tell me how to go about doing it.

Many thanks,

Edd

**Comment by andreynering** [comment, marked as answer]
Hi @eddgrant,

That error means that the `task` executable file wasn't found by your console. [How have you installed it?](https://taskfile.dev/#/installation)

Also, if `task` is available in the `$PATH`, you should probably type only `task`. `.\task` means your console will search for the binary in your current directory.

**Comment by eddgrant** [comment]
Hey @andreynering , thanks for getting back to me so quickly.

Turns out I've had a bit of a senior moment! I had completely neglected to remember that the binaries for task are O/S specific. I'm running Linux and had downloaded the appropriate Linux specific binary to the `bin` folder of my repository. When a colleague who runs Windows wanted to run the tasks I instructed them to use the executable in the `bin` folder! :man_facepalming: I now realise that the correct thing to do is to get them to install the Windows binary... 

I downloaded the binary to the `bin` folder of the repository really for convenience so that our CI hosts didn't have to have task already installed. However this will make it difficult to support multiple operating systems, so I will re-evaluate my choice of keeping a task binary in the repository and will ensure that everyone installs the O/S specific version to their path instead.

Thanks again,

Edd
